### PR TITLE
Simplify cleanup steps by avoiding type variance.

### PR DESCRIPTION
**Pull request by jan-ivar**
While reviewing, I found the cleanup steps a bit hard to follow. Hopefully this helps. @yutakahirano WDYT?

***
<a href="https://pr-preview.s3.amazonaws.com/jan-ivar/webtransport/pull/380.html" title="Last updated on Dec 7, 2021, 7:03 PM UTC (4c980a3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webtransport/380/3ddc1a8...jan-ivar:4c980a3.html" title="Last updated on Dec 7, 2021, 7:03 PM UTC (4c980a3)">Diff</a>

***
<a href="https://pr-preview.s3.amazonaws.com/jan-ivar/webtransport/pull/380.html" title="Last updated on Mar 15, 2022, 11:57 AM UTC (87252ab)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webtransport/380/e3c2cfc...jan-ivar:87252ab.html" title="Last updated on Mar 15, 2022, 11:57 AM UTC (87252ab)">Diff</a>

### CODE REVIEW

**Review by yutakahirano — APPROVED** [review]
LGTM, thank you!

File: index.bs [hunk 1]
```diff
@@ -717,10 +717,10 @@ To <dfn>initialize WebTransport over HTTP</dfn>, given a {{WebTransport}} object
...
    returns <b>blocked</b>, then abort the remaining steps and [=queue a network task=] with |transport|
    to run these steps:
-  1. If |transport|'s [=[[State]]=] is `"closed"` or `"failed"`, then abort these steps.
-  1. Let |error| be the result of [=WebTransportError/creating=] a {{WebTransportError}} with
-     `"session"`.
-  1. [=Cleanup=] |transport| with |error|, |error| and true.
+1. If |transport|'s [=[[State]]=] is `"closed"` or `"failed"`, then abort these steps.
+1. Let |error| be the result of [=WebTransportError/creating=] a {{WebTransportError}} with
+   `"session"`.
+1. [=Cleanup=] |transport| with |error|, |error| and true.
```
Comment (jan-ivar) [inline index.bs:723]:
```suggestion
  1. If |transport|'s [=[[State]]=] is `"closed"` or `"failed"`, then abort these steps.
  1. Let |error| be the result of [=WebTransportError/creating=] a {{WebTransportError}} with
     `"session"`.
  1. [=Cleanup=] |transport| with |error|, |error| and true.
```

Comment (jan-ivar) [reply index.bs:723]: Fix accidental unindent from merge
